implementation/choreography-to-orchestration/lambda/orchestration/UpdateReward.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+def lambda_handler(event, context):
+    # TODO implement
+    print ('Reward updated for the product id:', event['product_id'])
```

Comment:
Same here. Per your diagram, patterns/images/choreography-to-orchestration/orchestration.png, the Lambda stores data. Looks like the implementation is missing here

Reply:
Added Dynamodb table